### DISCUSSION TITLE
🎉북스테어즈 마이그레이션을 위한 Discussions가 오픈되었습니다!

### DISCUSSION THREAD

**Discussion by soryeongk** [TeamBookTez/nextjs-book-stairs#27] [Announcement]
리팩토링, 새로운 기술, 고민 등에 대한 내용을 이곳에 아카이빙합니다!
북스테어즈 팀원이 아니더라도 의견을 낼 수 있으며 모두가 부드러운 말투로 오픈 커뮤니케이션을 합니다!
대부분 정답은 정해져있지 않고, 우리는 서로를 존중합니다 :))

**Comment by Gyuminn** [comment]
신기하네요! 좋아요!
